src/libcore/num/f32.rs

```diff
@@ -80,75 +80,75 @@ pub mod consts {
 
     /// Archimedes' constant (π)
     #[stable(feature = "rust1", since = "1.0.0")]
-    pub const PI: f32 = 3.14159265358979323846264338327950288_f32;
+    pub const PI: f32 = 3.141_592_653_589_793_238_462_643_383_279_502_88_f32;
```

Comment:
It's arguable whether this is actually an improvement. If anyone wants to double-check this constant, they'll almost certainly be comparing against a number _without_ underscores, in which case the underscores get in the way. I doubt anyone will be directly reading the literals here either.